### PR TITLE
Stephanie Rodriguez

### PR DESCRIPTION
**Pull request by stephorozcobuilds**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,78 +1,79 @@
...
 Lesson 1<br>
-<a href="/intro-workbook/01LessonOne">Homework</a><br>
+<a href="./01LessonOne">Homework</a><br>
 <br><br>
 Lesson 2<br>
-<a href="/intro-workbook/02LessonTwo/in-class-project/">Class Project</a><br>
-<a href="/intro-workbook/02LessonTwo/homework/">Homework</a><br>
+<a href="./02LessonTwo/in-class-project/">Class Project</a><br>
+<a href="inclassproject.html">class webpage</a>
+<a href="./2LessonTwo/homework/">Homework</a><br>
```
Comment (victoriachuang) [inline index.html]: This path should be ./02LessonTwo!

Comment (stephorozcobuilds) [inline index.html]: awesome, the change worked. Thank you.

Comment (victoriachuang) [inline index.html]: Happy to help!

File: 02LessonTwo/homework/index.html [hunk 2]
```diff
@@ -1,10 +1,56 @@
...
+<h4>TV Shows I Like To Watch</h4>
+<!--figure tags necessary to add captions below images-->
+  <figure>
+    <img src="https://s-media-cache-ak0.pinimg.com/236x/a8/f1/f2/a8f1f29d528c08843fa07e6118f04d9f.jpg">
+      <figcaption>The Walking Dead</figcaption>
+  </figure>
+
+  <figure>
+      <img src="./empire.png">
+        <figcaption>Empire</empire>
```
Comment (victoriachuang) [inline 02LessonTwo/homework/index.html]: Careful! The closing tag on this should be /figcaption

Comment (stephorozcobuilds) [inline 02LessonTwo/homework/index.html]: Ah! Wow I must have been thinking about empire too much.

Comment (victoriachuang) [inline 02LessonTwo/homework/index.html]: Haha it happens

File: 05LessonFive/Checkpoint1/css/style.css [hunk 3]
```diff
@@ -1,3 +1,50 @@
...
+  text-align: center;
+  text-decoration: none;
+  margin: auto;  
+}
+
+.who {
+  background-color: rgb(240,240,240);
+  text-align: center;
+  width: 500px;
+  float:right;
```
Comment (victoriachuang) [inline 05LessonFive/Checkpoint1/css/style.css]: if you have two boxes next to each other, they should be floating in the same direction, or else they might not be aligned.

Comment (stephorozcobuilds) [inline 05LessonFive/Checkpoint1/css/style.css]: Ok, I will still try to play around with floating my divs correctly. I was just having trouble mastering that feature and just found that this worked the best for me.

File: 07LessonSeven/homework/css/style.css [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+.hire {
+  width: 450px;
+  height: 250px;
+  float: left;
+  background-color: green;
+  padding-left: 50px;
+  padding-top: 70px;
+}
+
+.job {
```
Comment (victoriachuang) [inline 07LessonSeven/homework/css/style.css:55]: can you combine with hire?

File: 08LessonEight/homework/css/exercise-06-style.css [hunk 5]
```diff
@@ -1 +1,52 @@
...
+
+.truck img {
+  width: 380px;
+}
+
+.plane img {
+  width: 380px;
+}
+
+.train img {
```
Comment (victoriachuang) [inline 08LessonEight/homework/css/exercise-06-style.css:28]: can you combine truck, plane and train imgs?

File: 15LessonFifteen/Checkpoint3/index.html [hunk 6]
```diff
@@ -1,10 +1,82 @@
...
+            <div class="vineyard">
+              <div class="curtain">
+                <h1>WINERY UVA</h1>
+                <h3>Without doubt, one of the most consistently high-quality wineries in the Barossa region"</h3>
+                <p>Pope Francis-Respected Sommelier</p>
+              </div>
+            </div>
+        </div>
 
+        <div class="container-3" "clearfix">
```
Comment (victoriachuang) [inline 15LessonFifteen/Checkpoint3/index.html]: this clearfix won't work! it's outside the class declaration

**Pull request closed without merging**